etna/datasets/internal_datasets.py

```diff
@@ -360,6 +360,46 @@ def read_data(path: Path, part: str) -> np.ndarray:
     df_test.to_csv(dataset_dir / f"traffic_2008_{dataset_freq.lower()}_test.csv.gz", index=True, compression="gzip")
 
 
+def get_traffic_2015_dataset(dataset_dir: Path) -> None:
+    """
+    Download and save traffic (2015-2016) dataset.
+
+    24 months worth of hourly data (24 daily records) that describes the occupancy rate, between 0 and 1, of different
+    car lanes of the San Francisco bay area freeways across time. Data was collected by 862 sensors from
+    Jan. 1st 2015 to Dec. 31th 2016. Dataset has prediction horizon: 24.
+
+    Notes
+    -----
+    There is another "traffic" dataset that is also popular and used in papers for time series tasks. This
```

Comment:
We should probably point out that we have this dataset in our library and give a name of it.